pkgs/development/python-modules/torch/source/default.nix

```diff
@@ -503,6 +525,7 @@ buildPythonPackage rec {
     which
     ninja
     pybind11
+    (python.withPackages (p: [ p.typing-extensions ]))
```

Comment:
This again should have been caused by the python leakage? Otherwise `typing-extensions` is already in `dependencies`.

Reply:
Indeed, I was able to remove it.